src/StackAdmin/Resources/Commands.ResourceManager/Cmdlets/Commands.Resources.Rest.csproj

```diff
@@ -20,7 +20,7 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\..\..\Stack\Debug\ResourceManager\AzureResourceManager\AzureRM.Resources\</OutputPath>
+    <OutputPath>$(StackPackageFolder)\Debug\ResourceManager\AzureResourceManager\AzureRM.Resources\</OutputPath>
```

Comment:
Can you revert this change? I think creating that StackPackageFolder might be what is failing the build (it is still failing after the latest change)